tests/integration/test_fim/test_synchronization/test_response_timeout.py

```diff
@@ -105,7 +105,7 @@ def purge_manager_db():
     def detect_synchronization_start(time_out=1):
         start_time = datetime.now()
         while datetime.now() < start_time + timedelta(seconds=time_out):
-            ssh_stdin, ssh_stdout, ssh_stderr = ssh.exec_command("sudo cat /var/ossec/logs/ossec.log")
+            ssh_stdin, ssh_stdout, ssh_stderr = ssh.exec_command("sudo cat /var/ossec/logs/wazuh.log")
```

Comment:
Same here.

Reply:
Fixed in https://github.com/wazuh/wazuh-qa/commit/20fb78d2a4218a84aa5ce2a4323ff3437686b23b